x-pack/plugins/triggers_actions_ui/public/application/components/add_message_variables.tsx

```diff
@@ -32,7 +32,7 @@ export const AddMessageVariables: React.FunctionComponent<Props> = ({
     messageVariables?.map((variable: ActionVariable, i: number) => (
       <EuiContextMenuItem
         key={variable.name}
-        data-test-subj={`variableMenuButton-${i}`}
+        data-test-subj={`variableMenuButton-${variable.name}`}
```

Comment:
This makes the underlying unit tests more reliable as `i` changes when you introduce new variables.